data/remnant outfits.txt

```diff
@@ -341,6 +341,20 @@ outfit "Quantum Key Stone"
 	"quantum keystone" 1
 	description "This precious artifact attunes a ship's quantum oscillations in a way that allows it to travel through certain otherwise impassible wormholes in the Ember Waste. The stones are rare and valuable, because they can only be mined from one location in the Waste."
 
+outfit "Salvage Scanner"
+	category "Systems"
+	licenses
+		Remnant
+	cost 410000
+	thumbnail "outfit/salvage scanner"
+	"mass" 7
+	"outfit space" -7
+	"outfit scan power" 13
+	"outfit scan speed" 1
+	"tactical scan power" 84
+	description "When the Remnant unraveled the alien point defence turrets guarding the vaults on Aventine, they also deciphered the mechanisms that guided the ancient weapons. After much research they transformed those guidance systems into important tools for analyzing alien ships."
+	description "	While all Remnant ships have built in outfit and tactical scanners in order to determine whether an enemy ship is worth engaging and looting, some captains still prefer to install external scanners in order to have superior scanning ranges."
```

Comment:
`"    While all Remnant ships are equipped with internal scanners, some captains still prefer to boost the range and power to more effectively select enemy ships worth targeting."`

Could also be `their range and power` instead of `the range and power`.

Reply:
I'm not convinced on this, but I think it is probably an improvement on mine, so adding it in.